projects/spectator/src/lib/test-framework.ts

```diff
@@ -0,0 +1,20 @@
+import { CustomMatcherFactory } from './matchers';
```

Comment:
I'm not sure about the name :)

Reply:
I know, any suggestions?

Reply:
I thought that I can locate it in `utils.ts` but I can't due to circular dependencies (`isRunningInJsDom` is used in matchers -> matchers used in `addMatchers`)

Reply:
maybe core.ts?

Reply:
Fine for me :)
Will push changes in few minutes